src/main/java/net/sf/jabref/exporter/IExportFormat.java

```diff
@@ -57,8 +58,7 @@
      *            should be exported. If null, all entries will be exported.
      * @throws Exception
      */
-    void performExport(BibDatabase database, MetaData metaData,
- String file, Charset encoding,
-            Set<String> entryIds) throws Exception;
+    void performExport(BibDatabase database, MetaData metaData, String file, Charset encoding, Set<String> entryIds,
+            JournalAbbreviationRepository repository) throws Exception;
```

Comment:
Yes, the ExportFormat class uses the Layout class which itself needs the JournalAbbreviation. Thus also the interface has to know about it... I don't like it since also all the other exports have now a dependency on JournalAbbreviation. Any idea how to solve it? Pass the JournalAbbreviationLoader to the constructor of ExportFormat?